libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -152,6 +152,13 @@ class BreezSDKMapper {
         }
     }
     
+    static func dictionaryOf(backupStatus: BackupStatus) -> [String: Any] {
```

Comment:
I think this should be `[String: Any?]` as lastBackupTime is optional/nullable

Reply:
You probably right although I didn't get any complain from the compiler and it worked.

Reply:
fixed